test/api/units/feedback_test.rb

```diff
@@ -25,4 +24,20 @@ def test_get_awaiting_feedback
       assert_json_matches_model response, expected, ['id']
     end
   end
+
+  def test_tasks_for_task_inbox
+    user = User.first
+    unit = Unit.first
+
+    expected_response = unit.tasks_for_task_inbox(user)
```

Comment:
Tests were incomplete and are on another branch yet to be pushed, I'll remove these from here (weren't meant to be in PR)